cmd/nvidia-ctk-installer/container/runtime/runtime.go

```diff
@@ -66,6 +66,12 @@ func Flags(opts *Options) []cli.Flag {
 			Destination: &opts.RestartMode,
 			EnvVars:     []string{"RUNTIME_RESTART_MODE"},
 		},
+		&cli.BoolFlag{
+			Name:        "enable-cdi-in-runtime",
+			Usage:       "Enable CDI in the configured runt	ime",
+			Destination: &opts.EnableCDI,
+			EnvVars:     []string{"RUNTIME_ENABLE_CDI"},
```

Comment:
nit: do we want to update the envvar to match the name of the flag, `ENABLE_CDI_IN_RUNTIME`? I don't have a strong opinion though since the other options all start with the `RUNTIME_` prefix.

Reply:
I purposfully chose this to match the other `RUNTIME_` envvars here.